mock/py/mockbuild/util.py

```diff
@@ -1339,6 +1339,9 @@ def check_config(config_opts):
 
 @traceLog()
 def include(config_file, config_opts, is_statement=False):
+    if config_file.endswith('.tpl') and not os.path.isabs(config_file):
```

Comment:
What will happen with `include('fedora-rawhide-x86_64.cfg')`?  I'm just not sure the `.tpl` condition is correct.

Reply:
I agree that we should allow *.cfg as well

Reply:
Alright, np. Done.